### PR TITLE
Add last hooks to hydrates

### PR DESCRIPTION
**Pull request by ggrbill**

### CODE REVIEW

File: alfasim_sdk/hook_specs.py [hunk 1]
```diff
@@ -44,6 +44,53 @@ def compute_source_terms(
...
+def calculate_slurry_viscosity(
+    ctx: "void*",
+    alpha_f: "void*",
+    mu_f: "void*",
+    mu_f_layer: "void*",
+) -> "int":
+    """
+    Internal simulator hook to calculate slurry viscosity of layer(s).
+
+    :param ctx: ALFAsim's plugins context
```
Comment (nicoddemus) [inline alfasim_sdk/hook_specs.py]: It is not clear which of those parameters are input and which are output...

File: alfasim_sdk/hook_specs.py [hunk 2]
```diff
@@ -44,6 +44,53 @@ def compute_source_terms(
...
+    d_disp_fields: "void*",
+    P: "void*",
+    rho_f: "void*",
+    mu_f: "void*",
+    sin_theta_f: "void*",
+    delta_x_f: "void*",
+) -> "int":
+    """
+    Internal simulator hook to calculate slip velocity between fluids
+    and solids.
```
Comment (nicoddemus) [inline alfasim_sdk/hook_specs.py]: It is not clear which of those parameters are input and which are output...

File: alfasim_sdk/hook_specs.py (hunk 1 above)
Comment (ggrbill) [reply alfasim_sdk/hook_specs.py]: I will add extra explanation to docstring

File: alfasim_sdk/hook_specs.py (hunk 2 above)
Comment (ggrbill) [reply alfasim_sdk/hook_specs.py]: I will add extra explanation to docstring

**Pull request merged**